row_067_ID.py

```
@@ -0,0 +1,109 @@
+	
+@app.route('/uploader', methods = ['POST'])
+def upload_file():
+   if request.method == 'POST':
+       if 'file' not in request.files:
+           return "No file"
+       else:
+           f = request.files['file']
+           filename = secure_filename(f.filename)           
+           #Save to passporteye folder to process
+           os.chdir("C:/Users/shahi/AppData/Local/Programs/Python/Python36-32/Lib/site-packages/passporteye")
+           file_path="C:/Users/shahi/AppData/Local/Programs/Python/Python36-32/Lib/site-packages/passporteye/"+filename
```

Ruff (0.13.2)

Local variable `file_path` is assigned to but never used

Remove assignment to unused variable `file_path`

(F841)